boards/arm/hexiwear_k64/pinmux.c

```diff
@@ -59,8 +59,7 @@ static int hexiwear_k64_pinmux_init(struct device *dev)
 	struct device *gpiob =
 	       device_get_binding(DT_NXP_KINETIS_GPIO_GPIO_B_LABEL);
 
-	gpio_pin_configure(gpiob, 12, GPIO_DIR_OUT);
-	gpio_pin_write(gpiob, 12, 0);
+	gpio_pin_configure(gpiob, 12, GPIO_OUTPUT_ACTIVE | GPIO_ACTIVE_LOW);
```

Comment:
If we don't care about logical levels then it's better to use `GPIO_OUTPUT_LOW`. The flag will configure the pin as output and initialize it to physical low output level.

Reply:
Done